docs/deployment/kyuubi_on_kubernetes.md

```diff
@@ -0,0 +1,98 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+<div align=center>
+
+![](../imgs/kyuubi_logo.png)
+
+</div>
+
+# Deploy Kyuubi On Kubernetes
+
+## Requirements
+
+If you want to deploy Kyuubi on Kubernetes, you'd better get a sense of the following things.
+
+* Use Kyuubi official docker image or build Kyuubi docker image
+* An active Kubernetes cluster
+* Reading About [Deploy Kyuubi engines on Kubernetes](https://kyuubi.readthedocs.io/en/latest/deployment/egine_on_kubernetes.html)
```

Comment:
use relative path if possiable, otherwise please link to kyuubi.apache.org instead of kyuubi.readthedocs.io

Reply:
OK, i will fix this.